test/test_utils.py

```
@@ -14,7 +14,7 @@
 from pathlib import Path
 from textwrap import dedent
 
-from conda_build import api, metadata
+from conda_build import api, metadata, exceptions
```

Ruff

`conda_build.api` imported but unused

Remove unused import: `conda_build.api`

(F401)